### PR TITLE
add notebook for Chapter 4

### PR DESCRIPTION
**Pull request by TomKealy**
This PR adds a notebook for Chapter 4 of Bayes Rules!

### CODE REVIEW

**Comment by canyon289** [comment]
Thanks @TomKealy, theres two notebooks here, which one is the correct one to review?

**Comment by canyon289** [comment]
Ive merged the two formats that were proposed into one one main. Please be sure to follow that. From yours I really liked the local file reads.

One thing to note be sure you're using pymc (not pymc3) and that your environment doesnt have theano in it. We're about to move to pymc soon and it would great to ensure these note books don't become oudated immediately

**Comment by canyon289** [comment]
Sounds good! Ill be sure to review these one extra thoroughly then to give you as much feedback as possible.

Super appreciate your efforts

**Comment by canyon289** [comment]
@TomKealy Apologies on the delay, I'll get to reviewing them this weekend

File: Bayes_Rules/Chapter 4 Balance and Sequentiality in Bayesian Analyses.ipynb [hunk 1]
```diff
@@ -0,0 +1,818 @@
+{
```
Comment (canyon289) [inline Bayes_Rules/Chapter 4 Balance and Sequentiality in Bayesian Analyses.ipynb:1]: <p>Add Chapter Name as header</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc_resources/pull/188/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Comment (canyon289) [inline Bayes_Rules/Chapter 4 Balance and Sequentiality in Bayesian Analyses.ipynb:1]: <p>IIs something supposed to be here?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc_resources/pull/188/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>

File: Bayes_Rules/Chapter_4_Balance_and_Sequentiality_in_Bayesian_Analyses.ipynb [hunk 2]
```diff
@@ -0,0 +1,817 @@
+{
```
Comment (canyon289) [inline Bayes_Rules/Chapter_4_Balance_and_Sequentiality_in_Bayesian_Analyses.ipynb:1]: <p>i believe this file can be removed</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc_resources/pull/188/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

**Comment by canyon289** [comment]
@TomKealy any update on these notebooks?

**Comment by TomKealy** [comment]
Hi @canyon289! I am working on these comments today. Sorry, have been off work ill.

**Comment by canyon289** [comment]
Hey,
No problem! Just want to make sure youre supported

**Comment by canyon289** [comment]
@TomKealy it seems Ci is broken, My apologies on that Ill fix it this weekend for you. Thank you for continuing to support this!

**Pull request merged**